exp/wsj/README.md

```diff
@@ -50,6 +50,7 @@ Check that `$LVSR` environment variable points to this repository and
    model option, beam size and subset there are alignment images and 
    `report.txt` which contains transcripts, approximate CER and WER and other 
    auxiliary information.
+   <part> specifies the dataset to decode. For example it can take on the value 'valid'
```

Comment:
Or 'test', it should be more useful to people who want to repeat the experiment. + full stop.